app/views/dashboard.mustache

```diff
@@ -78,32 +78,34 @@
                 <div class="box-section">
                   <h4 class="heading-small">{{#__}}dashboard.progression{{/__}}</h4>
 
-                  <div class="grid-row">
```

Comment:
columns (below, line 89) need to be wrapped in a grid-row for the layout css to behave predictably